pyproject.toml

```diff
@@ -14,6 +14,7 @@ requests = "^2.24.0"
 cryptography = "^3.4.8"
 certifi = "*"
 pycryptodome = "^3.10.1"
+pyopenssl = "21.0.0"
```

Comment:
Might be able to remove this if `cryptography` cert parsing is possible
```suggestion
```

Reply:
I did the opposite. removed cryptography since pyopenssl has it as a dependency.
Pyopenssl is necessary to read information(dates) of the certificate.

Reply:
done